vault/templates.topic.config.property.md

```diff
@@ -0,0 +1,20 @@
+---
+id: xUh9GJgBLYk5HoEHLCopP
+title: Property
+desc: 'Template for individual configuration property documents.'
+updated: 1645168284443
+created: 1645167115254
+---
+
```

Comment:
I'm wondering whether we should standardize on what the first header of these notes should be?

- I think we could have a first header, and use note ref embed modifications (this example presupposes the first header is `properties`):
  - `![[assetsPrefix|dendron.topic.publish.config.assetsPrefix#properties,1]]`
- The benefit here is that we'd be able to:
  - Optionally link directly to just the first header
  - Optionally embed just the first header in notes
  - Optionally rename/move the header section using refactoring commands in the future if a pruning/repotting session resulted in any updates
- I think it may be a good idea for us to have a note standard where we always have a header at the top of the note in order to benefit from refactoring commands we may use in the future as otherwise the top section is orphaned in a way

Reply:
currently, the convention is to have the first header be `## Summary`. we can either use that or use `## Metadata` to indicate the difference here

Reply:
Let's defer this discussion to the main pruning / repotting task since this won't affect the docs that will be pushed here.

Reply:
linked this comment in [[Config Documentation Prune and Repot|dendron://private/task.2022.02.16.config-documentation-prune-and-repot]]